src/services/dashboards/widgets/_components/WidgetFrame.vue

```diff
@@ -108,22 +119,38 @@ export default defineComponent<Props>({
         },
         currency: {
             type: String as PropType<Currency>,
-            default: CURRENCY.USD,
+            default: undefined,
         },
     },
-    setup(props, { emit }:SetupContext) {
+    setup(props) {
+        const specialDate = (startDate) => {
+            // custom date range
+            const diff = dayjs().diff(dayjs(startDate), 'day', true);
+            if (diff < 1) return 'Today';
+            if (diff >= 6 && diff < 7) return 'Past 7 days';
+            return dayjs(startDate).format('YY-MM');
+        };
```

Comment:
(This function is an example.)
Returns the value specified when the `end` of `props.dateRage` does not exist, such as this code.

Reply:
dayjs library supports relative time format  :)
Also, when using relative time format, language must be considered.
That means, return value of specialDate function must be reactive to i18n.locale.
There is a hook which name is i18nDayjs(probably) to cover these feature.